zipline/pipeline/filters/filter.py

```diff
@@ -533,3 +535,34 @@ class StaticAssets(StaticSids):
     def __new__(cls, assets):
         sids = frozenset(asset.sid for asset in assets)
         return super(StaticAssets, cls).__new__(cls, sids)
+
+
+class NoMissingValues(CustomFilter, SingleInputMixin):
```

Comment:
I tend to try to name filters after a positive property rather than the negation of a property, e.g., `AllPresent` instead of `NoMissingValues`.  The main reason for that preference that I think the positive assertion is easier to parse if it gets negated: `~AllPresent()` ("not all present") is easier for me to think about than `~NoMissingValues` ("not no missing values").  Thoughts?

Reply:
`CustomFilter` already has `PositiveWindowLengthMixin`, I'll add `StandardOutputs`.

I agree, a positive assertion is is more easily read if negated.